api/v1alpha1/servicebinding_types.go

```diff
@@ -46,7 +92,12 @@ type ServiceBinding struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	Spec   ServiceBindingSpec   `json:"spec,omitempty"`
+	// +kubebuilder:validation:Required
+	// +required
+	Spec ServiceBindingSpec `json:"spec,omitempty"`
+
+	// Observed status of the service binding within the namespace
+	// +optional
 	Status ServiceBindingStatus `json:"status,omitempty"`
```

Comment:
If I'm not wrong, `omitempty` takes precedence on `// +optiona`, `// +required` and `// +kubebuilder:validation:Required` are the same.  I'd say we can remove the required and optional comments for now.

```suggestion
	Spec ServiceBindingSpec `json:"spec,omitempty"`

	// Observed status of the service binding within the namespace
	Status ServiceBindingStatus `json:"status,omitempty"`
```